<?xml version="1.0"?>
<launch>
    <arg name="waypoints_file"     default="$(find-pkg-share waypoint_navigation)/waypoints_cfg/waypoints.yaml"/>
    <arg name="use_sim_time"       default="false"/>
    <!--
    <arg name="StartFromTheMiddle" default="false"/>
    <arg name="cmd_vel_topic_in"   default="/move_base/cmd_vel"/>
    <arg name="cmd_vel_topic_out"  default="/wp_nav/cmd_vel"/>
    <arg name="robot_frame"        default="base_footprint"/>
    <arg name="max_vel_param"      default="/move_base/TrajectoryPlannerROS/max_vel_x"/>
    <arg name="min_vel_param"      default="/move_base/TrajectoryPlannerROS/min_vel_x"/>
    <arg name="min_dist_err"       default="0.3"/>
    <arg name="min_yaw_err"        default="0.3"/>
    <arg name="costmaps_file"      default="$(find waypoint_navigation)/scripts/switch_costmaps.yaml"/>
    <arg name="tandem_scan"        default="scan"/>
    <arg name="use_angle"          default="20"/>
    <arg name="danger_dist"        default="1.0"/>
    -->

    <!-- Set use_sim_time of following nodes to True -->
    <set_parameter name="use_sim_time" value="$(var use_sim_time)"/>


    <node pkg="waypoint_navigation" exec="waypoint_nav" output="screen">
        <param name="waypoints_file"           value="$(var waypoints_file)"/>
    </node>

</launch>